Assets/MixedRealityToolkit-SDK/Inspectors/Utilities/Solvers/InBetweenInspector.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.SDK.Utilities.Solvers;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Inspectors.Utilities.Solvers
+{
+    [CustomEditor(typeof(InBetween))]
+    public class InBetweenEditor : Editor
+    {
+        private SerializedProperty trackedObjectProperty;
+        private SerializedProperty transformTargetProperty;
+        private InBetween solverInBetween;
+
+        private static readonly string[] fieldsToExclude = new string[] { "m_Script" };
```

Comment:
Why do we need this if we're already getting the serialized properties?  I don't see us calling the `base.OnInspectorGUI`

Reply:
Line 43:
DrawPropertiesExcluding(serializedObject, fieldsToExclude);